tests/Umbraco.Tests.UnitTests/Umbraco.Core/DeliveryApi/PublishedContentCacheTests.cs

```diff
@@ -239,6 +239,117 @@ public void PublishedContentCache_DenyListIsCaseInsensitive()
         Assert.IsNull(content);
     }
 
+    [TestCase(true)]
+    [TestCase(false)]
+    public void PublishedContentCache_GetById_SupportsAllowList(bool allowed)
+    {
+        var allowList = allowed ? new[] { "theOtherContentType" } : new[] { "someOtherType" };
+        var publishedContentCache = new ApiPublishedContentCache(CreateRequestPreviewService(), CreateDeliveryApiSettings(allowedContentTypeAliases: allowList), CreateApiDocumentUrlService(), _contentCache, CreateVariationContextAccessor());
```

Comment:
Looks like this could be moved into a private helper method called `CreateApiPublishedContentCache` and then used from all the tests. You would pass in an instance of `IOptionsMonitor<DeliveryApiSettings>` that you have created via `ApiPublishedContentCache`.